src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -128,28 +128,62 @@
  * </li>
  * </ul>
  * <p>
+ * Note, that files with the same path and base name but which have different
+ * extensions will be considered as files that belong to different resource bundles.
+ * </p>
+ * <p>
  * To configure the check to check only files which have '.properties' and
  * '.translations' extensions:
  * </p>
  * <pre>
  * &lt;module name="Translation"&gt;
  *   &lt;property name="fileExtensions" value="properties, translations"/&gt;
+ *   &lt;property name="requiredTranslations" value="fr"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Note, that files with the same path and base name but which have different
- * extensions will be considered as files that belong to different resource bundles.
+ * Example:
  * </p>
+ * <pre>
+ * #messages.properties
+ * hello=Hello
+ * cancel=Cancel
+ *
+ * #messages.translations
+ * hello=Hallo
+ * ok=OK
+ * </pre>
+ * <pre>
+ * messages.properties: Properties file 'messages_fr.properties' is missing.
+ * messages.translations: Properties file 'messages_fr.translations' is missing.
+ * </pre>
  * <p>
  * An example of how to configure the check to validate only bundles which base
  * names start with "ButtonLabels":
  * </p>
  * <pre>
  * &lt;module name="Translation"&gt;
  *   &lt;property name="baseName" value="^ButtonLabels.*$"/&gt;
+ *   &lt;property name="requiredTranslations" value="fr"/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * #ButtonLabels.properties
+ * save=Save
+ * cancel=Cancel
+ *
+ * #ButtonLabels.translations
```

Comment:
Make it ButtonLabels_fr.properties

To show ability to catch missed translation.

Reply:
After changing it to ```ButtonLabels_fr.properties``` and ```ButtonLabels_fr.translation```
I am getting
```
hp@DESKTOP-OPBFIIB MINGW64 ~/Desktop
$ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
      "-//Checkstyle//DTD Check Configuration 1.3//EN"
      "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="Translation">
        <property name="fileExtensions" value="properties, translations"/>
        <property name="baseName" value="^ButtonLabels.*$"/>
        <property name="requiredTranslations" value="fr"/>
  </module>
</module>
hp@DESKTOP-OPBFIIB MINGW64 ~/Desktop
$ cat ButtonLabels_fr.properties
hello=Bonjour
name=Nom
hp@DESKTOP-OPBFIIB MINGW64 ~/Desktop
$ cat ButtonLabels_fr.translations
hello=Bonjour
cancel=Annuler
hp@DESKTOP-OPBFIIB MINGW64 ~/Desktop
$  java -jar checkstyle-10.9.3-SNAPSHOT-all.jar -c config.xml ButtonLabels_fr.properties ButtonLabels_fr.translations
Starting audit...
[ERROR] C:\Users\hp\Desktop:1: Properties file 'ButtonLabels.translations' is missing. [Translation]
[ERROR] C:\Users\hp\Desktop:1: Properties file 'ButtonLabels.properties' is missing. [Translation]
Audit done.
Checkstyle ends with 2 errors.
```

Reply:
Rename file to ButtonLabels_fr.properties

Reply:
`.translation` is only for one example.

Reply:
Done